CMakeLists.txt

```diff
@@ -164,4 +171,6 @@ target_include_directories(cppwinrt PRIVATE "${winmd_SOURCE_DIR}")
 
 
 include(CTest)
-add_subdirectory(test)
+if(BUILD_TESTING)
```

Comment:
Can you confirm it still includes testing by default?

Reply:
Yes, the option is created by the CTest module and is `ON` by default: https://cmake.org/cmake/help/latest/module/CTest.html